tests/tests_shared.c

```diff
@@ -215,12 +215,12 @@ bool parse_cmd_line(int argc, char** argv, const sir_cl_arg* args, size_t num_ar
     return true;
 }
 
-void wait_for_keypress(void) {
-    (void)printf(WHITEB(EMPH("press any key to exit...")) "\n");
+void wait_for_keypress(void) { // GCOVR_EXCL_START
```

Comment:
Why do I recall we had already resolved this? You wrote a script that called it with --wait and then killed it or something like that

Reply:
Fixed